modules/highgui/src/window.cpp

```diff
@@ -481,12 +486,12 @@ int cv::createButton(const String&, ButtonCallback, void*, int , bool )
 
 #endif
 
-#if   defined (HAVE_WIN32UI)  // see window_w32.cpp
-#elif defined (HAVE_GTK)      // see window_gtk.cpp
-#elif defined (HAVE_COCOA)    // see window_carbon.cpp
+#if   defined (HAVE_WIN32UI)                    // see window_w32.cpp
+#elif defined (HAVE_GTK)                        // see window_gtk.cpp
+#elif defined (HAVE_COCOA)                      // see window_carbon.cpp
 #elif defined (HAVE_CARBON)
-#elif defined (HAVE_QT)       // see window_QT.cpp
-#elif defined (WINRT) && !defined (WINRT_8_0) // see window_winrt.cpp
+#elif defined (HAVE_QT)                         // see window_QT.cpp
+#elif defined (WINRT) && !defined (WINRT_8_0)   // see window_winrt.cpp
```

Comment:
A [few more lines](https://github.com/Itseez/opencv/pull/4155/files#diff-5068c0e1d1253df0f3183ded2585e52fR68), new ```elif``` clause islonger than others and comments have been aligned so I kept the alighnment using stronger indentation. Anyway this can easily be reverted.
Done.